release.sh

```diff
@@ -5,6 +5,38 @@ cd $DIR
 
 set -e
 
+git remote update
+
+upstream=${1:-'@{u}'}
+remote=$(git rev-parse "$upstream")
+local=$(git rev-parse @)
```

Comment:
could you put some comments on the usage of `@` and `@{u}`?

Reply:
I made this a little bit more concise, but yes I'll comment whats going on here.